tests/roles/keystone_adoption/tasks/main.yaml

```diff
@@ -68,3 +68,22 @@
     for service in cinderv3 glance neutron nova placement swift; do
       ${BASH_ALIASES[openstack]} service list | awk "/ $service /{ print \$2; }" | xargs ${BASH_ALIASES[openstack]} service delete || true
     done
+
```

Comment:
We should have per-service task files in the pull_openstack_configuration role instead of having OS Diff snippets in the service roles?

To match the documentation flow, we want to execute OS Diff before the EDPM adoption step. Users more familiar with Adoption may want to run it in different order than the docs are written, but i think the test suite should stick with the planned docs order.